src/docs/delightful-developer-metrics/ui.mdx

```diff
@@ -9,14 +9,9 @@ In order to plot your metrics, we have developed a prototype user interface, whi
 
 ## Enabling the UI
 
-The first step towards using the UI is to enable it, since right now it’s feature flagged. 
-These two flags need to be turned on:
-
-- `organizations:custom-metrics` - ingestion and UI
-- `organizations:ddm-ui` - UI
-- `organizations:metric-meta` - sample events and code locations
-- `organizations:use-metrics-layer-in-alerts` - custom metric alerts
+The first step towards using the UI is to enable it, since right now it’s feature flagged. The following feature flag is needed:
 
+- `organizations:custom-metrics`
```

Comment:
Is there a reason for leaving out the second feature flag needed for ingestion? 

organizations:custom-metrics-experimental

Reply:
as far as I can tell, the `custom-metrics-experimental` flag is not used for anything right now